src/Microsoft.Azure.SignalR.Common/Endpoints/ServiceEndpointManagerBase.cs

```diff
@@ -177,8 +183,8 @@ private IReadOnlyList<HubServiceEndpoint> UpdateAndGetRemovedHubServiceEndpoints
         private HubServiceEndpoint CreateHubServiceEndpoint(string hub, ServiceEndpoint endpoint, bool needScaleTcs = false)
         {
             var provider = GetEndpointProvider(endpoint);
-
-            return new HubServiceEndpoint(hub, provider, endpoint, needScaleTcs);
+            var hubEndpoint = new HubServiceEndpoint(hub, provider, endpoint, needScaleTcs);
```

Comment:
You can just recover this line to `return new HubServiceEndpoint(hub, provider, endpoint, needScaleTcs);`